contracts/MedianOracle.sol

```diff
@@ -15,9 +14,7 @@ interface IOracle {
  * @notice Provides a value onchain that's aggregated from a whitelisted set of
  *         providers.
  */
-contract MedianOracle is Ownable, IOracle {
-    using SafeMath for uint256;
-
+contract MedianOracle is OwnableUpgradeable, IOracle {
```

Comment:
This is more than a compiler version upgrade, since it makes the oracle upgradeable also... Seems worthy of a separate discussion. I can see some reasons add it, but we've so far kept upgradeability segmented to the places where it was needed most at the time.

Reply:
This actually does not make the contract upgradable in practice. We just use the oz "upgradable" dependencies. This give us the flexibility to deploy it as either upgradable (through a proxy) or immutable through a deployment configuration. (also the oz scripts complain if you have both upgradable and non upgradable libraries in the same project)

When we deploy the oracle contract, we sill perform the immutable deployment.

Reply:
Got it, thanks